src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(FigureColor.values().length);
+        return FigureColor.values()[index].toString();
```

Comment:
Use the `name()` method instead of `toString()` for getting the String representation of enum constants to ensure you get the exact name of the enum constant as defined in the enum declaration.